docs/quick.md

```diff
@@ -233,6 +233,7 @@ follwing steps:
    parameters:
      cas-type: cstor
      cstorPoolCluster: cstor-storage
```

Comment:
can we add the comment here saying , name needs to be changed accroding to the name has been used to configure cspc ?